build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -127,6 +127,13 @@ hostGateway: {{ .Values.hostGateway | quote }}
 #
 trafficEncapMode: {{ .Values.trafficEncapMode | quote }}
 
+# The aggregated cluster-wide Pod CIDRs (not the per-Node Pod CIDRs).
+# This field is applicable only in networkPolicyOnly mode and ignored in other traffic modes. It can be left empty,
+# but it is recommended to specify the correct Pod CIDRs string, as Traceflow may not work correctly for inter-Node
+# Pod-to-Pod traffic otherwise.
+# Example: "10.10.0.0/16" for IPv4-only, or "10.10.0.0/16,fd00::/12" for dual-stack.
+podCIDRs: ""
```

Comment:
I would also add this to the Helm values.

Reply:
Done